torch_geometric/datasets/actor.py

```diff
@@ -50,7 +50,7 @@ class Actor(InMemoryDataset):
     def __init__(self, root: str, transform: Optional[Callable] = None,
                  pre_transform: Optional[Callable] = None):
         super().__init__(root, transform, pre_transform)
-        self.data, self.slices = torch.load(self.processed_paths[0])
+        self.load(self.processed_paths[0], data_cls=Data)
```

Comment:
nit: do we need to specify `data_cls`?
```suggestion
        self.load(self.processed_paths[0])
```

Reply:
Good catch! Let's drop it.